actions/websocket_actions.test.jsx

```diff
@@ -845,18 +845,29 @@ describe('handlePluginEnabled/handlePluginDisabled', () => {
             const mockComponent = 'mockRootComponent';
             registery.registerRootComponent(mockComponent);
 
-            const dispatchArg = store.dispatch.mock.calls[0][0];
+            let dispatchArg = store.dispatch.mock.calls[0][0];
```

Comment:
Are this test changes still needed?

Reply:
Should we add tests for the new events?

Reply:
@larkox I think the test adds value so I decided to keep it. Yes the new functionality should have tests. Thanks for suggesting this